var/spack/repos/builtin/packages/herwig3/package.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import os
+
+
+class Herwig3(AutotoolsPackage):
+    """Herwig is a multi-purpose particle physics event generator."""
+
+    homepage = "https://herwig.hepforge.org"
+    url      = "https://herwig.hepforge.org/downloads/Herwig-7.2.1.tar.bz2"
+
+    version('7.2.1', sha256='d4fff32f21c5c08a4b2e563c476b079859c2c8e3b78d853a8a60da96d5eea686',
+            url='http://lcgpackages.web.cern.ch/lcgpackages/tarFiles/sources/MCGeneratorsTarFiles/Herwig-7.2.1.tar.bz2')
+
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool',  type='build')
+    depends_on('m4',       type='build')
+    # DEPENDS lhapdf Boost Python GSL thepeg fastjet vbfnlo openloops madgraph5amc njet gosam automake
+    depends_on('lhapdf',   type=('build', 'run'))
+    depends_on('thepeg@2.2.1', when='@7.2.1', type=('build', 'link', 'run'))
+    depends_on('boost', type='build')
+    depends_on('python', type=('build', 'run'))
+    depends_on('gsl', type=('build', 'run'))
+    depends_on('fastjet', type=('build', 'run'))
+    depends_on('vbfnlo@3:', type=('build', 'run'))
+    depends_on('madgraph5amc', type=('build', 'run'))
+    depends_on('njet', type=('build', 'run'))
+    # depends_on('gosam', type=('build', 'run'))
+    depends_on('gosam-contrib', type=('build', 'run'))
+    depends_on('openloops', type=('build', 'run'))
```

Comment:
type=build/run is usually only used for things like Python that don't link to libraries. I assume the default deptype is what you want.